pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -365,6 +365,7 @@ private void initPolicesCache(SystemTopicClient.Reader<PulsarEvent> reader, Comp
             if (hasMore) {
                 reader.readNextAsync().thenAccept(msg -> {
                     refreshTopicPoliciesCache(msg);
+                    notifyListener(msg);
```

Comment:
It's best to wait for initPolicesCache to complete before notifying the listeners.

Reply:
make sense to me, I will fix it